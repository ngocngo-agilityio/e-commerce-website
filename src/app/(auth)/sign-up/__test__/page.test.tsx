// Libs
import { render } from '@testing-library/react';

// Sections
import SignInPage from '../page';

jest.mock('@sections', () => ({
  SignUpForm: () => <div>SignUpForm</div>,
}));

describe('SignInPage page', () => {
  afterEach(() => {
    jest.clearAllMocks();
  });

  it('Should render match with snapshot.', () => {
    const { container } = render(<SignInPage />);

    expect(container).toMatchSnapshot();
  });
});
