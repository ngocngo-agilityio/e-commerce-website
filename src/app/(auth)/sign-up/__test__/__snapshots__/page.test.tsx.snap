// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInPage page Should render match with snapshot. 1`] = `
<div>
  <h2
    class="chakra-heading css-g0bx3n"
  >
    Get Started Now
  </h2>
  <div>
    SignUpForm
  </div>
  <p
    class="chakra-text css-qbqnjq"
  >
    Have an account?Â 
    <span
      class="chakra-text css-1kz23e"
    >
      <a
        href="/sign-in"
      >
        Sign In
      </a>
    </span>
  </p>
</div>
`;
